modules/apiserversource-yaml.adoc

```diff
@@ -6,14 +6,14 @@
 [id="apiserversource-yaml_context"]
 = Creating an API server source by using YAML files
 
-You can use the following procedure to create an API server source by using YAML files.
+Creating Knative resources by using YAML files uses a declarative API, which enables you to describe event sources declaratively and in a reproducible manner. To create an API server source by using YAML, you must create a YAML file that defines an `ApiServerSource` object, then apply it by using the `oc apply` command.
```

Comment:
Not quite sure how to fix it, but files uses seems awkward.

_Creating Knative resources by using YAML files enables you
When you create Knative resources by using YAML files, you use a declarative API ??_

Would the second sentence be a stronger lead?

Reply:
responded to similar comment in other PR, gonna leave as is for now